woocommerce/wc-admin
# Update chart colors to match spec

Fixes #1705.

This PR:
- Updates the constants used to generate the chart colors to match what is defined in #375.
- Updates the chart legend to call `getColors` only on visible items.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/53749777-6ddae280-3ea8-11e9-93e5-27b17eb9ced5.png)

### Detailed test instructions:
- Go to the _Products_ report and active the _Top Products by Items Sold_ filter, so there are at least 5 items displayed in the chart.
- Verify colors match #375.
- Deactivate other items one by one and verify colors still match #375.
